app/controllers/Swiss.scala

```diff
@@ -249,9 +250,18 @@ final class Swiss(
   def player(id: String, userId: String) =
     Action.async {
       WithSwiss(id) { swiss =>
-        env.swiss.api.playerInfo(swiss, userId) flatMap {
+        env.swiss.api.playerInfo(swiss, userId).flatMap {
           _.fold(notFoundJson()) { player =>
-            JsonOk(fuccess(lila.swiss.SwissJson.playerJsonExt(swiss, player)))
+            if (swiss.settings.isMultiPoint)
+              env.swiss.api.playerToPairingGames(player).flatMap { pairingsWithGames =>
+                JsonOk(fuccess(lila.swiss.SwissJson.playerJsonExtMultiPoint(swiss, player, pairingsWithGames)))
```

Comment:
I think we should refactor to have just one swissJson call, and have pairingWithGames an optional input.